source/engine/audio/NcAudioImpl.cpp

```diff
@@ -38,18 +39,16 @@ struct NcAudioStub : public nc::audio::NcAudio
 
 namespace nc::audio
 {
-auto BuildAudioModule(bool enableModule, Registry* reg) -> std::unique_ptr<NcAudio>
+auto BuildAudioModule(const config::AudioSettings& settings, Registry* reg) -> std::unique_ptr<NcAudio>
 {
-    if(enableModule)
+    if(settings.enabled)
     {
         NC_LOG_TRACE("Creating NcAudio module");
```

Comment:
This is a very minor nit but - will you change "Creating" to "Building" in the trace so that it aligns with the method name? Here and for the other NcXImpls

Reply:
changed